### ISSUE TITLE
CSU fcrepo data transfer to S3

### ISSUE THREAD

**Issue by aprilrieger** [scientist-softserv/scholarworks#26]
Ref #25 

This is the third part of work needing to be accomplished for Serverless IIIF implementation for CSU Hyrax([SOW](https://docs.google.com/document/d/1hVD2QW2tvyR0r9YpqEw7kTYNydmCiFYkxX5wRDBYBwI/edit))

Steps to complete:
- [x] copying the files fedora transfer to the s3 bucket (details below).
- [x] reconfiguring/restarting Fedora

Bucket:
region: `us-west-2`
name: `hyrax-fcrepo-cache`
AWS direct [link](https://us-west-2.console.aws.amazon.com/s3/buckets/hyrax-fcrepo-cache?region=us-west-2&bucketType=general&tab=objects) once logged in under csu role

Lambda Function:
region: `us-west-2`
function url: `https://6cff3iwnwia62qiqak74dkfsku0fwjsd.lambda-url.us-west-2.on.aws/`
AWS direct [link](https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/functions) once logged in under csu role

**Comment by aprilrieger** [comment]
Rob started transferring the data form fedora to s3 today.

**Comment by aprilrieger** [comment]
The files have been transferred to s3 and we are ready to schedule maintenance. I have reached out to the client in slack.
We have reconfigured the deployment to use the updated bucket and are ready to apply the changes of the updated config to use s3 bucket and then re-sync fcrepo/s3 bucket to transfer the remaining files from the date of the transfer till now and test that the changes have not broken the links to the files to download.

We also need to ensure that there are no imports running.


During the 30min - 1 hour maintenance window with client the remaining work to complete is:
- [x] Disable imports
- [x] Restart Fedora with the new config that targets the s3 bucket
```
sudo su
cd /etc/tomcat/conf.d/
cp fedora4.conf fedora4.conf.bak <--- already done
cp fedora4.conf.withs3 fedora4.conf
systemctl restart tomcat
```
- [x] Test that downloads of files are not broken for this: https://scholarworks.calstate.edu/concern/theses/jw827m11w < should work right away after upgrade
- [x] Re-sync fcrepo to s3 to capture anything imported form the date we transferred the initial data till now
```
sudo su
cd /opt/fcrepo-data
sudo su
cd fcrepo.binary.directory
irb
load "ruby fcrepo-copy.rb"
```
- [x] Test that we can download the file here:  https://scholarworks.calstate.edu/concern/theses/np193j55s < should work only after s3 sync
